game/kernel/jak1/kscheme.cpp

```diff
@@ -264,7 +264,11 @@ u64 make_string_from_c(const char* c_str) {
 }
 
 extern "C" {
+#ifdef __APPLE__
+void _arg_call_linux() asm ("_arg_call_linux");
+#else
 void _arg_call_linux();
+#endif
```

Comment:
Without `asm ("_arg_call_linux")`, the object file will reference `__arg_call_linux` (two leading underscores) on macOS. Let me know if there is some way other than this you prefer.